evap/staff/views.py

```diff
@@ -216,7 +217,9 @@ class Stats:
                 stats.first_start = min(stats.first_start, evaluation.vote_start_datetime)
                 stats.last_end = max(stats.last_end, evaluation.vote_end_date)
     degree_stats = OrderedDict(sorted(degree_stats.items(), key=lambda x: x[0].order))
-    degree_stats["total"] = total_stats
+
+    degree_stats_with_total: Dict[Union[Degree, str], Stats] = cast(Dict[Union[Degree, str], Stats], degree_stats)
```

Comment:
The type is correctly assigned after the cast:
```suggestion
    degree_stats_with_total = cast(Dict[Union[Degree, str], Stats], degree_stats)
```